rasa/core/constants.py

```diff
@@ -65,3 +65,5 @@
 
 # RabbitMQ message property header added to events published using `rasa export`
 RASA_EXPORT_PROCESS_ID_HEADER_NAME = "rasa-export-process-id"
+
+CIRCUIT_BREAKER_TRIPPED = "circuit breaker tripped"
```

Comment:
I'm not sure if this is a proper constant, or a user-facing string. I can see you have only used it in one place, and it appears it eventually will be shown to the user along other action names. In this case I thing it's better to treat it as a regular string, that would imply removing this variable and putting the string directly in `_collect_action_executed_predictions`.